Framework/DataHandling/src/LoadILLReflectometry.cpp

```diff
@@ -1,22 +1,181 @@
-/*WIKI*
- TODO: Enter a full wiki-markup description of your algorithm here. You can then
- use the Build/wiki_maker.py script to generate your full wiki page.
- *WIKI*/
-
 #include "MantidDataHandling/LoadILLReflectometry.h"
+
 #include "MantidAPI/Axis.h"
 #include "MantidAPI/FileProperty.h"
+#include "MantidAPI/FunctionFactory.h"
+#include "MantidAPI/IPeakFunction.h"
 #include "MantidAPI/MatrixWorkspace.h"
 #include "MantidAPI/RegisterFileLoader.h"
+#include "MantidAPI/SpectrumInfo.h"
 #include "MantidAPI/WorkspaceFactory.h"
+#include "MantidDataObjects/TableWorkspace.h"
+#include "MantidHistogramData/LinearGenerator.h"
+#include "MantidKernel/BoundedValidator.h"
+#include "MantidKernel/EnabledWhenProperty.h"
+#include "MantidKernel/ListValidator.h"
 #include "MantidKernel/OptionalBool.h"
 #include "MantidKernel/Quat.h"
 #include "MantidKernel/UnitFactory.h"
+#include "MantidDataObjects/Workspace2D.h"
+#include "MantidDataObjects/WorkspaceCreation.h"
+
+namespace {
+/// A struct for information needed for detector angle calibration.
+struct DirectBeamMeasurement {
+  double detectorAngle;
+  double detectorDistance;
+  double fittedPeakCentre;
+  double positionOfMaximum;
+};
+
```

Comment:
Could any of the helper functions below also go into the anonymous namespace?

Reply:
Which helper functions? The anonymous namespace continues all the way to line 178 and should include all the functions 'below'. Did you mean some private method of `LoadILLReflectometry`? I think all of them use a member variable or the logger but maybe I missed something?

Reply:
Oh whoops, my bad - got my curly braces mixed up. I presumably meant all the helper functions already in the anonymous namespace